lib/matplotlib/contour.py

```diff
@@ -1409,64 +1410,38 @@ def find_nearest_contour(self, x, y, indices=None, pixel=True):
 
         Returns
         -------
-        contour : `.Collection`
-            The contour that is closest to ``(x, y)``.
+        path : int
```

Comment:
This is a pretty big API change....

Reply:
It isn't an API change.  As noted by @anntzer at https://github.com/matplotlib/matplotlib/issues/27070#issuecomment-1761789486, the docstring was previously wrong.

Reply:
oh, never mind then!  Sorry for the noise.